homeassistant/components/hunterdouglas_powerview/__init__.py

```diff
@@ -56,7 +56,7 @@
 
 CONFIG_SCHEMA = cv.removed(DOMAIN, raise_if_present=False)
 
-PLATFORMS = [Platform.COVER, Platform.SCENE, Platform.SENSOR]
+PLATFORMS = [Platform.COVER, Platform.SCENE, Platform.SENSOR, Platform.BUTTON]
```

Comment:
Please sort these in ABC order.
```suggestion
PLATFORMS = [Platform.BUTTON, Platform.COVER, Platform.SCENE, Platform.SENSOR]
```